### DISCUSSION TITLE
Weird "focus" style on package pages with Chrome

### DISCUSSION THREAD

**Discussion by targos** [npm/feedback#298] [Website] [answered]
I can reproduce with Google Chrome or Edge:

Just go to a package page, for example https://www.npmjs.com/package/npm and click anwhere inside the rendered README:

![image](https://user-images.githubusercontent.com/2352663/112327006-594b2c00-8cb5-11eb-8741-897cd8ad80e8.png)

**Comment by MylesBorins** [comment]
@targos thanks for reporting! I've made an issue internally and am escalating to the appropriate engineering team.

**Comment by targos** [comment]
Just noticed there's the same thing in the keywords section:
![image](https://user-images.githubusercontent.com/2352663/112329025-1be79e00-8cb7-11eb-8669-701f4d061337.png)

**Comment by lumaxis** [comment, marked as answer]
Thanks everyone for bringing this to our attention! This should be corrected since a few hours ago thanks to a fix by @BAJ- 🙂
